activity.py

```diff
@@ -380,12 +380,12 @@ def __open_on_journal(self, widget, url):
             'icon-color': profile.get_color().to_string(),
             'mime_type': 'text/uri-list',
         }
-        for k, v in metadata.items():
+        for k, v in list(metadata.items()):
             jobject.metadata[k] = v
         file_path = os.path.join(get_activity_root(), 'instance',
                                  '%i_' % time.time())
         open(file_path, 'w').write(url + '\r\n')
-        os.chmod(file_path, 0755)
+        os.chmod(file_path, 0o755)
```

Comment:
@pro-panda @quozl  python 3 will interpret "0o755" as octal and ultimately 755 as 493. This is a discrepancy that 2to3 creates. We should change "0o755" manually to "0755" or "755".

Reply:
I don't understand what you mean.  Octal is how we usually understand and explain file modes, as they are a bit mask of permissions.  In python3 "0755" correctly gives us _SyntaxError: invalid token_.  Decimal 755 would give an insane file mode.  I'm really glad we don't have to use seven OR'd constants!

Reply:
Thanks for clarifying.